README.md

```diff
@@ -1,15 +1,10 @@
 # ChemDraw Add-ins
+
 Welcome to ChemDraw add-ins! This repository contains documentation and examples for writing your own ChemDraw add-ins.
 
 ## Prerequisites
 
-### Windows
-
-You must install [ChemOffice Professional](http://www.perkinelmer.com/product/chemoffice-professional-chemofficepro) 17.1 or above to install and run the add-in examples.
-
-### Mac OS X
-
-You must install [ChemDraw Professional](http://www.perkinelmer.com/product/chemdraw-professional-chemdrawpro) 17.1 or above to install and run the add-in examples.
+You must install [ChemOffice](https://revvitysignals.com/products/research/chemdraw) 17.1 or above to install and run the add-in examples.
```

Comment:
According to the datasheet only ChemOffice and above supports add-ins, and ChemOffice has both win and mac versions. Thus removed the instruction for different platforms. https://image.info.revvitysignals.com/lib/fe3211747364047c761777/m/1/5e8b16cf-91bd-456e-9394-a79d3df77eff.pdf?elqTrackId=9c53448a38964278b21d3f4578683e2d&elqaid=10141&elqat=2&_gl=1*1kssuzy*_ga*MzcwMTIwMDMuMTcwNTMyMzc5NQ..*_ga_F7XGE57X1J*MTcwNTMyMzc5NS4xLjEuMTcwNTMyMzk5My41OS4wLjEwNzA4MTExMTU.

Reply:
I don't think ChemOffice exists now. You might just want to say to install ChemDraw and mention that not all ChemDraw product levels support add-ins?

Reply:
Updated. Please review again.